js/interface.js

```diff
@@ -1288,7 +1423,61 @@ $('.file-manager-wrapper')
     }
 
     $searchTerm.keyup();
+  })
+  .on('dragstart', '.file-row', function(e) {
+    var dragingItem = $(e.target).data();
+    e.originalEvent.dataTransfer.setData('text', JSON.stringify(dragingItem));
+    $('.panel-title.list-holder').addClass('drop-area');
+    $('.app-holder').addClass('drop-area');
+    $('.file-row[data-file-type="folder"]').each(function() {
+      var item = $(this);
+      if (dragingItem.id != item[0].dataset.id && !$(item).hasClass('active')) {
+        $(item).addClass('drop-area');
+      }
+    });
+    $('.bread-link').addClass('drop-area').last().removeClass('drop-area');
+  })
+  .on('dragend', function(e) {
+    $('.panel-title.list-holder').removeClass('drop-area');
+    $('.app-holder').removeClass('drop-area');
+    $('.file-row[data-file-type="folder"]').removeClass('drop-area');
+    $('.bread-link').removeClass('drop-area');
+  })
+  .on('drop', '.drop-area', function(e) {
+    e.preventDefault();
+    var area = $(this);
+    var dropArea = area.data();
+    var items = $('.file-row.active');
+    var itemType = JSON.parse(e.originalEvent.dataTransfer.getData('text'));
+    var $element = $('.file-row[data-id=' + itemType.id + ']');
+    var checkDraggedItems = checkDraggedFileIfSelected(itemType, items);
+    var alertConfirmationMovingItem = confirm('Are you sure you want to move item(s)?');
```

Comment:
`Are you sure you want to move these item(s)?`

Reply:
fixed